contracts/src/system/contracts/TenSystemCalls.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.28;
+
+import "@openzeppelin/contracts/proxy/utils/Initializable.sol";
+
+contract TenSystemCalls is Initializable {
```

Comment:
how would people know the address of this contract?
We need to bubble it up somehow and publish it in the "known addresses" thingy